test/y2storage/planned/lvm_lv_test.rb

```diff
@@ -27,12 +27,42 @@
 describe Y2Storage::Planned::LvmLv do
   using Y2Storage::Refinements::SizeCasts
 
-  subject(:lvm_lv) { described_class.new("/") }
+  subject(:lvm_lv) { described_class.new(mount_point) }
+
+  let(:mount_point) { "/" }
 
   let(:volume_group) do
     instance_double(Y2Storage::LvmVg, size: 30.GiB, extent_size: 4.MiB)
   end
 
+  describe "#initialize" do
+    context "when creating a planned lv for a given mount point" do
+      context "and the mount point is '/'" do
+        let(:mount_point) { "/" }
+
+        it "sets lv name to 'root'" do
+          expect(subject.logical_volume_name).to eq("root")
+        end
+      end
+
+      context "and the mount point is not '/'" do
+        let(:mount_point) { "/var/lib/docker" }
+
+        it "sets lv name based on mount point" do
+          expect(subject.logical_volume_name).to eq("var_lib_docker")
+        end
+      end
+    end
+
+    context "when creating a planned lv without mount point" do
+      let(:mount_point) { nil }
+
+      it "does not set lv name" do
```

Comment:
This makes me think. Proposing volumes without a mount point (to be used raw by some database or something like that) is a real use case that is coming just around the corner. With our current implementation it will work if only one of such volumes is specified (it will receive the "lv" name thanks to the `LvmCreator` fallback). A more robust solution is needed (either introduce support for the LV name in control.xml or improve the `LvmCreator` mechanism). We need a PBI to track this.

@jsrain Do you think it makes sense to introduce a `lv_name` attribute in control.xml (I'm all for it)? The alternative is to use things like "var_lib_docker" for LVs with a mount point (already implemented) and "lv0", "lv1", etc. for volumes without mount point (right now, only "lv" without the number, so the second unmounted volume would explode).

Reply:
Out of the scope of this PR, anyways :-)

Reply:
Hmm, I just remembered we have a mechanism to avoid duplicated LV names, even if we didn't add the conflicting LV. So having several non-mounted LVs will not fail with the current implementation. The non-mounted LVs will be successfully created with names like "lv", "lv0", "lv1"...

So this is not as bad/urgent as I originally feared. But still one question stands - does it make sense to add a `lv_name` tag to the volume specification in control.xml? With that, release managers could get nicer names than `var_lib_docker` or `lv`.